src/Stryker.Core/Stryker.Core/Reporters/JsonReporter/JsonReportFileComponent.cs

```diff
@@ -22,7 +22,7 @@ private JsonReportFileComponent(string language, string source, ISet<JsonMutant>
 
         public JsonReportFileComponent(FileLeaf file, ILogger logger = null)
         {
-            var log = logger ?? ApplicationLogging.LoggerFactory.CreateLogger<JsonReportFileComponent>();
+            logger = logger ?? ApplicationLogging.LoggerFactory.CreateLogger<JsonReportFileComponent>();
```

Comment:
Did sonarqube not mention that you should not overwrite a method parameter value?

Reply:
No

Reply:
Mhmm 🤔